app/src/main/java/org/fossasia/susi/ai/helper/PrefManager.java

```diff
@@ -29,6 +32,21 @@ private static SharedPreferences getPreferences() {
                 .getApplicationContext());
     }
 
+    public PrefManager(Context context) {
+        int PRIVATE_MODE = 0;
+        Context _context = context;
+        pref = _context.getSharedPreferences(PREF_NAME, PRIVATE_MODE);
+        editor = pref.edit();
+    }
+
+    public void setFirstTimeLaunch(boolean isFirstTime) {
+        editor.putBoolean(IS_FIRST_TIME_LAUNCH, isFirstTime);
```

Comment:
You don't need this imo.  use the below defined method to put boolean in preference. And similarly to get boolean.
```
public static void putBoolean(String preferenceKey, boolean preferenceValue) {
         getPreferences().edit().putBoolean(preferenceKey, preferenceValue).apply();
}
```

Reply:
@chiragw15  Done, Please check now.